env0/provider_test.go

```diff
@@ -3,55 +3,48 @@ package env0
 import (
 	"context"
 	"fmt"
+	"os"
+	"strings"
+	"testing"
+
 	"github.com/env0/terraform-provider-env0/client"
 	"github.com/env0/terraform-provider-env0/utils"
 	"github.com/golang/mock/gomock"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-	"os"
-	"strings"
-	"testing"
 )
 
-var (
-	apiClientMock *client.MockApiClientInterface
-	ctrl          *gomock.Controller
-)
-
-var testUnitProviders = map[string]func() (*schema.Provider, error){
-	"env0": func() (*schema.Provider, error) {
-		provider := Provider("")()
-		provider.ConfigureContextFunc = func(ctx context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
-			return apiClientMock, nil
-		}
-		return provider, nil
-	},
-}
-
 func runUnitTest(t *testing.T, testCase resource.TestCase, mockFunc func(mockFunc *client.MockApiClientInterface)) {
+	os.Setenv("TF_ACC", "1")
```

Comment:
`TF_ACC` is required when running terraform acceptance tests.
(Will error if not set).
In addition, this actually uncovered several bugs. When this env variable is set, some errors turn into panics.